x/programs/runtime/util_test.go

```diff
@@ -20,12 +20,14 @@ type testRuntime struct {
 	DefaultGas uint64
 }
 
-func (t *testRuntime) CallProgram(program codec.Address, actor codec.Address, function string, params ...interface{}) ([]byte, error) {
+func (t *testRuntime) CallProgram(program codec.Address, actor codec.Address, height uint64, timestamp uint64, function string, params ...interface{}) ([]byte, error) {
```

Comment:
Let's not add these and then use the CallContext instead.

Reply:
+1

Reply:
Replaced fields by a `Context` instead.